tests/test_hexbytes.py

```diff
@@ -0,0 +1,45 @@
+from ethpm_types import BaseModel, HexBytes
+
+TEST_MODEL_DICT = {
```

Comment:
Should we include a more filled out `HexBytes` value in here? Like an address?

Reply:
If you think so? I was just trying to show that I didn't break the functionality of HexBytes or Pydantic by adding the validation and serialization methods.

Reply:
Ah, this seems good then. I suppose that would not change anything.